src/pages/guides/application_state.md

```diff
@@ -33,29 +32,116 @@ NAME=Joe
+No pre-configuration is required, just install the libraries and use them in your project. We will be transparently using your AppBuilder credentials to authorize and entitle your requests.
 
-The state library is meant for storing and accessing small values, while the files library should be used for storing bigger amounts of data.
+*When should I use Files vs State?*
 
-To learn more or to try them out, please visit the following GitHub repositories:
-- [Adobe I/O File Storage library](https://github.com/adobe/aio-lib-files)
-- [Adobe I/O Key/Value Storage library](https://github.com/adobe/aio-lib-state)
+- Files is good for bandwidth and State is good for latency.
+- Files supports sharing data via presigned-url, State supports setting expirations.
+- As a rule of thumb if you expect your data to grow larger than 100KBs go with Files, otherwise use State.
 
-After reviewing each of the libraries, you may have noticed that the sample code only requires your Runtime namespace credentials in order to start accessing the cloud services behind the scenes. This is handled through the [Adobe I/O Token Vending Machine](https://github.com/adobe/aio-tvm) (TVM). TVM is a set of Adobe I/O Runtime actions exposed as an API that allows developers to trade their credentials for temporary and restricted tokens to external cloud services. Users authenticate to the TVM with their Adobe I/O Runtime (OpenWhisk) credentials and are only authorized to access their own resources.
+## State
 
-You can also opt out of using Runtime (OpenWhisk) and leverage your own cloud services (for example, Azure). Please see the sample code in either of the library GitHub repositories for more information.
+***We are introducing major changes for State**, now hosting our storage service. The documentation refers to the new version which is still in **developer preview**. Documentation for the last stable version based on CosmosDB is available [here](https://github.com/adobe/aio-lib-state/tree/3.x)*
+
+***How is my data stored?***
+
+- State is a multi-tenant storage, your data is isolated in a "State container" mapping to the I/O Runtime namespace and application Workspace.
+- Each region stores your data independently, treat it as a different instance. We support `amer` and `emea`. *`apac` is coming soon.*
+- Note the default time-to-live for a key-value which is 1 day, the maximum is 1 year (365 days).
+
+### Getting started
+
+***Library usage, from an I/O Runtime Action:***
+
+```bash
+npm install @adobe/aio-lib-state@next
+```
+
+```js
+  const stateLib = require('@adobe/aio-lib-state')
+
+  // init with implicit I/O Runtime credentials, default region is 'amer'.
+  const state = await stateLib.init()
+  // set an explicit region
+  const state2 = await stateLib.init({ region: 'emea' })
```

Comment:
Do we define regions somewhere? What regions are available? Why would I choose one over another?

Reply:
is the `how is my data stored` intro enough for now? 
https://adobedocs.github.io/app-builder/guides/application_state/#state

Reply:
will add something more on the bullet point